docs/guide/writing/style-guides.md

```diff
@@ -1,72 +1,87 @@
 # Style Guides
 
-A style guide contains a set of standards for writing and designing content. It helps maintain a consistent style, voice, and tone across your documentation, whether you're a lone writer or part of a huge docs team. It can provide guidelines for different documentation deliverables, such as API reference manuals, tutorials, release notes, or overviews of complex technical concepts.
-
-A style guide saves documentarians time and trouble by providing a single reference for writing about common topics, features, and more. 
-The guidelines in a style guide help writers to produce documentation that has the same tone and grammatical style, regardless of who writes the documentation.
-A consistent tone and style can make your content easier to read by reducing your users' [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) and increasing their confidence in the content's authority. 
+A style guide contains a set of standards for writing and designing content. It
+helps maintain a consistent style, voice, and tone across your documentation,
+whether you're a lone writer or part of a huge docs team. A style guide saves
+documentarians time and trouble by providing a single reference for writing
+about common topics, features, and more. It can provide guidelines for different
+documentation deliverables, such as API reference manuals, tutorials, release
+notes, or overviews of complex technical concepts.
+
+A consistent tone and style makes your content easier to read, reducing your
+users' [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) and
+increasing their confidence in the content's authority.
 
 ## Write your own style guide?
 
-A style guide can be something as simple as a list of decisions you've made about how to refer to different items you frequently write about. Or it can be as complicated as the mighty tomes of major publication houses. 
-
-You can certainly create a style guide of your own. For the sake of simplicity, this approach might work if you're a lone writer or just starting a small docs group. But neither software nor its documentation operates in a vacuum, so it's a good idea to consult other resources as well. Working from an existing style guide can also help you figure out which things matter in your style guide.
+You can certainly create a style guide of your own. This approach might work if
+you're a lone writer or just starting a small docs group. But neither software
+nor its documentation operates in a vacuum, so it's a good idea to consult other
+resources as well. Working from an existing style guide can also help you figure
```

Comment:
```suggestion
resources as well. Working from an existing style guide also helps you figure
```

Reply:
I made this edit just to play with inserting a suggestion. @barriebyron taught me about this feature today at Write the Docs Writing Day. I'll use this all the time at work now! Curious to see exactly how it works.